docs/Project.toml

```
@@ -12,10 +11,11 @@ Manifolds = "1cead3c2-87b3-11e9-0ccd-23c62b72b94e"
 ManifoldsBase = "3362f125-f0bb-47a3-aa74-596ffd7ef2fb"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Pluto = "c3e4b0f8-55cb-11ea-2926-15256bba5781"
+PlutoStaticHTML = "359b1769-a58e-495b-9770-312e911026ad"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-Documenter = "0.24"
-Manifolds = "0.5"
+Documenter = "0.27"
+Manifolds = "0.5,0.6,0.7"
 ManifoldsBase = "0.11,0.12"
+PlutoStaticHTML = "2.1"
```

```suggestion
PlutoStaticHTML = "3.4"
```
CI is running v2.1.1. I have no idea why you were locally on a newer version

---

Ah, I even checked status locally but not the toml - thanks for noticing.